src/app/work-item/work-item-detail/typeahead-dropdown/typeahead-dropdown.component.scss

```diff
@@ -2,106 +2,69 @@
 
 .typeahead-dropdown {
   padding-left: 0;
-    
-    .neutral-entry {
-      font-style: italic !important;
-      opacity: 0.5 !important;
-    }
-
-    .no-value-label {
-      font-style: italic !important;
-      opacity: 0.5 !important;      
-    }
-
-    .details-dropdown {
-      /* width: 100%; */
-    }
-
-    .btn-bar {
-      margin-left: 5px;
-      span {
-        z-index: 11;
-      }
-    }
-
-    .dropdown-search {
-      position: relative;
-      /* width: 40%; */
-      height: 1px;
-      z-index: 10000;
-
-      .list-container {
-        min-width: em(200);
-        z-index: 10;
-        background-color: $color-pf-black-100;
-        border: 1px solid $color-pf-black-500;
-
-        .dropdown-search-icon {
-          position: absolute;
-          line-height: em(30);
-          padding-left: em(5);
-          height: inherit;
-          color: $color-pf-black-500;
+  .neutral-entry {
+    font-style: italic !important;
+    color: $color-pf-blue;
+  }
+  .no-value-label {
+    font-style: italic !important;
+    color: $color-pf-blue;
+  }
+  .details-dropdown {
+    /* width: 100%; */
+  }
+  .item-ul{
+    position: absolute;
+    top: 100%;
+    left: 0;
+    z-index: 1000;
```

Comment:
Will this cause conflicts? Can the default z-index be used from the dropdown component?